examples/with-federated-authentication/src/pages/index.tsx

```diff
@@ -0,0 +1,162 @@
+import Image from "next/image";
+import styles from "./index.module.css";
+import { getWebAuthnAttestation, TurnkeyApi } from "@turnkey/http";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { buffer } from "stream/consumers";
+import { useState } from "react";
+
+type subOrgFormData = {
+  orgName: string;
+};
+
+type privateKeyFormData = {
+  privateKeyName: string;
+};
+
+const es256 = -7;
```

Comment:
this probably deserves a comment + a link to the registry which defines the algorithm constants (IANA Cose identifier): https://www.iana.org/assignments/cose/cose.xhtml